app/components/subcases/subcase-header.js

```diff
@@ -105,6 +105,11 @@ export default class SubcasesSubcaseHeaderComponent extends Component {
   *proposeForAgenda(meeting) {
     this.isAssigningToOtherAgenda = false;
     this.isLoading = true;
+    let decisionActivity = yield this.store.queryOne('decision-activity', {
+      'filter[subcase][:id:]': this.args.subcase.id,
+    });
+    decisionActivity.secretary = meeting.secretary;
```

Comment:
This throws `Uncaught TypeError: Cannot set properties of null (setting 'secretary')` when the secretary of the `meeting` is not set, which can happen for any meeting that was created before these changes.

Reply:
It complains because decisionActivity is `null`, which makes sense, since this `subcase` hasn't been proposed for an `agenda` yet. 

Not sure when `decisionActivity` might ever not be null actually... 

I'd be inclined to remove this added code block completely.